src/SingleCellAnnotator.imjoy.html

```diff
@@ -947,6 +947,9 @@
       if(this.current_image_data.exclusive_label === type){
         this.current_image_data.exclusive_label = false;
         this.current_image_data.location_labels = Object.assign({}, this.current_image_data.saved_location_label)
+        if((this.current_task_name ==='segmentation')&&(type==='Discarded')){
```

Comment:
this is to send masks to backend if 'Discard' is deselected. Note that in the current master, if user draws masks first and then deselect 'Discard', it will send masks and set labels as null. So this changes is consistent with that as well

Reply:
I think perhaps a better way to handle this is to disable mask annotation when Discarded is activated for an image. When the user click discard, I would disable the mask annotation, drop the changed mask, and send only the Discarded label to the backend.

Reply:
either way works. then I will delete the lines I added here

Reply:
removed